test/document_stores/test_elasticsearch.py

```diff
@@ -376,6 +382,22 @@ def test_write_documents_req_for_each_batch(self, mocked_document_store, documen
             mocked_document_store.write_documents(documents)
             assert mocked_bulk.call_count == 5
 
+    @pytest.mark.unit
+    def test_get_vector_similarity_query(self, mocked_document_store):
+        vec_sim_query = mocked_document_store._get_vector_similarity_query(np.random.rand(3).astype(np.float32), 10)
+        assert vec_sim_query["script_score"]["script"]["source"] == "dotProduct(params.query_vector,'embedding') + 1000"
+
+    @pytest.mark.unit
+    def test_get_vector_similarity_query_es_7_5_and_below(self, mocked_document_store):
+        # Patch server version to be 7.5.0
```

Comment:
Yeah here too, just a bit more motivation for why are we testing this, short comment?